sdk/lib/crt/math/arm/__64tof_worker.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * PROJECT:     ReactOS CRT library
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     Implementation of __i64tod
+ * COPYRIGHT:   Copyright 2021 Roman Masanin <36927roma@gmail.com>
+ */
+
+#ifdef _USE_64_BITS_
+typedef double FLOAT_TYPE;
+typedef unsigned long long FINT_TYPE;
+#define FRACTION_LEN 52
+#define EXPONENT_LEN 11
+#else
+typedef float FLOAT_TYPE;
+typedef unsigned int FINT_TYPE;
+#define FRACTION_LEN 23
+#define EXPONENT_LEN 8
+#endif
+
+typedef
+#ifndef _USE_SIGNED_
+unsigned
+#endif
+long long INT64SU;
+
+typedef union _FLOAT_RESULT
+{
+    FLOAT_TYPE value;
+    FINT_TYPE raw;
+} FLOAT_RESULT;
+
+#define SIGN_MASK 0x8000000000000000ULL
+
+#define EXPONENT_ZERO ((1 << (EXPONENT_LEN - 1)) - 1)
+
+#define NEGATE(x) (~(x) + 1)
+
+FLOAT_TYPE
+__64tof_worker(INT64SU value)
```

Comment:
Please rename the function to just `__64tof`, the same way as I previously renamed `__fto64` (because it's not called from assembly anymore).
```suggestion
__64tof(INT64SU value)
```